mmr-lib/Utils/SpoilerUtils.cs

```diff
@@ -54,7 +54,7 @@ public static void CreateSpoilerLog(RandomizedResult randomized, SettingsObject
                 }),
             };
 
-            if (settings.GenerateHTMLLog)
+            if (settings.OutputHTMLSpoiler)
```

Comment:
Can we make it so checking both checkboxes will output both spoiler logs?

Reply:
yes. I must have missed this section of code.